evm/src/memory/segments.rs

```diff
@@ -54,4 +54,19 @@ impl Segment {
             Segment::RlpRaw => "SEGMENT_RLP_RAW",
         }
     }
+
+    #[allow(dead_code)]
+    pub(crate) fn bit_range(&self) -> usize {
+        match self {
+            Segment::Code => 8,
+            Segment::Stack => 256,
+            Segment::MainMemory => 8,
+            Segment::Calldata => 8,
+            Segment::Returndata => 8,
+            Segment::Metadata => 8,
+            Segment::KernelGeneral => 8,
+            Segment::TxnData => 8,
```

Comment:
I think these three should be allowed to go up to 256 bits. The rest look good!

Reply:
Thanks!